templates/vagrant/Vagrantfile.erb

```diff
@@ -14,7 +14,12 @@ Vagrant.configure("2") do |config|
   # This disables up, provision, reload, and destroy for managed servers. Use
   # `vagrant orchestrate push` to communicate with managed servers.
   config.orchestrate.filter_managed_commands = true
-
+  <% if creds_prompt -%>
+  config.orchestrate.credentials do |creds|
```

Comment:
I think this is worth making a one liner unless there is other stuff that you're planning to put in the bootstrapping.